src/UNL/VisitorChat/Conversation/ConfirmationEmail.php

```diff
@@ -23,8 +23,29 @@ public static function sendConversation(\UNL\VisitorChat\Conversation\Record $co
         
         $email = new $class($conversation, $to, $fromId, $options);
         
+        $email->setReplyTo($email->getReplyTo());
```

Comment:
This seems a bit awkward in the traditional sense of a getter and setter--the getter doesn't actually relate to the setter. Other than that, it seems okay.